src/background/electronConfig.ts

```diff
@@ -2,15 +2,19 @@ import { join } from "path";
 import fs from "fs";
 import { app, dialog, shell } from "electron";
 import log from "electron-log";
-import { Config } from "@/infrastructures/Config";
+import { BaseConfig } from "@/infrastructures/Config";
 import { ConfigType } from "@/type/preload";
 
-export class ElectronConfig extends Config {
-  public configExists(): boolean {
+export class ElectronConfig extends BaseConfig {
+  getVersion() {
```

Comment:
老婆心かもなのですが、configの中にもバージョンが入っていてそっちと勘違いしちゃうかも･･･？
こうですかね。
```suggestion
  getAppVersion() {
```